test_unit.py

```diff
@@ -403,6 +405,102 @@ def test_serialize_keyreg_nonpart(self):
             "TEIAn70nYsCPhsWua/bdenqQHeZnXXUOB+jFx2mGR9tuH9pHR5cGWma2V5cmVn")
         self.assertEqual(golden, encoding.msgpack_encode(signed_txn))
 
+    def test_serialize_onlinekeyreg(self):
```

Comment:
Note that this test is identical to `test_serialize_keyreg_online`. The only difference is that this uses `OnlineKeyregTxn`. The reason for this is that in the wild, `KeyregTxn` might still be directly called so we want this test to still catch potential bugs that might be introduced in future.